quint/io-cli-tests.md

```diff
@@ -621,13 +621,28 @@ exit $exit_code
     ok mintSendTest passed 10000 test(s)
 
   2 passing (duration)
+[]
 [
-  "minter",
-  "balances"
-]
-[
-  "minter",
-  "balances"
+  [
+    "alice",
+    0
+  ],
+  [
+    "bob",
+    0
+  ],
+  [
+    "charlie",
+    0
+  ],
+  [
+    "eve",
+    0
+  ],
+  [
+    "null",
```

Comment:
I don't think this null should be here

Reply:
"null" is an account name here. This is probably confusing :))